packages/did-provider-ebsi/src/types.ts

```diff
@@ -43,5 +42,26 @@ export interface ICreateIdentifierArgs {
   kms?: string
   alias?: string
   type?: EbsiDidSpecInfo
-  options?: IKeyOpts
+  options?: {
+    methodSpecificId?: string // method specific id for import
+    secp256k1?: IKeyOpts | VerificationMethod
```

Comment:
I rather see the word Key in there. So secp256k1Key. Also what is the reason for the union. The VerificiationMethod interface extends IKey, which makes the name poor BTW (should be changed). You could just as well have added the purposes property as an optional prop to the IKey interface

Reply:
For the sake of consistency I just copied snippets from `did-provider-ion`. I cannot add anything to the `IKey` interface because it comes from `@veramo/core`, but I could make `IKeyOpts` extend it directly and keeping the required `privateKeyHex` property.